docs/readthedocs/source/doc/Ray/Overview/ray.md

```diff
@@ -116,3 +116,27 @@ Add the environment variables when calling `init_orca_context` would resolve the
 ```python
 sc = init_orca_context(cluster_mode, init_ray_on_spark=True, env={"LANG": "C.UTF-8", "LC_ALL": "C.UTF-8"})
 ```
+
+---
+### **5. FAQ**
+- **ValueError: Ray component worker_ports is trying to use a port number ... that is used by other components.**
+
+  This error is because that some port in worker port list is occupied by other processes. To handle this issue, you can set range of the worker port list by using parameters `min-worker-port` and `max-worker-port` in `init_orca_context()` as follows:
+
+  ```python
+  init_orca_context(extra_params={"min-worker-port": "30000", "max-worker-port": "30033"})
+  ```
+
+- **ValueError: Failed to bind to 0.0.0.0:8265 because it's already occupied. You can use `ray start --dashboard-port ...` or `ray.init(dashboard_port=...)` to select a different port.**
+
+  This error is because that ray dashboard port is occupied by other processes. To handle this issues, you can end the process occupyied the port or you can set the ray dashboard port by using parameters `dashboard-port` in `init_orca_context()` as follows:
```

Comment:
issues -> issue
occupyied -> that occupies
set -> manually set

Reply:
parameters -> parameter

Reply:
fixed.